soccer/src/soccer/soccer_mom/soccer_mom.cpp

```diff
@@ -0,0 +1,22 @@
+#include "soccer_mom.hpp"
+
+namespace tutorial {
+
+SoccerMom::SoccerMom() : Node("soccer_mom") {
```

Comment:
Again, I would add comments on what you are making so it's easier to follow what you are making.